### ISSUE TITLE
Version 1 Final Draft Editorial Meeting

### ISSUE THREAD

**Issue by drphilmarshall** [LSSTScienceCollaborations/ObservingStrategy#293]
Dear Chapter editors: @bethwillman @connolly
@ivezic @yoachim @davidtrilling @rhiannonlynne @akvivas @willclarkson @AshishMahabal @lmwalkowicz @knutago @dnidever @ebellm @fedhere @ohadshemmer @tanguita @MichelleLochner @egawiser @StephenRidgway @knutago

Thanks for all your efforts on your science sections, including your metrics, this spring! Our [timeline](https://github.com/LSSTScienceCollaborations/ObservingStrategy/milestones) from #264 has April 28th as our deadline for preparing our "Final Draft", consisting of a paper that is ready for internal review by us all. Let's meet up later this week to check in with our progress, and iron out various logistical issues before we start our late April sprint. As you saw, there has been some interesting discussion about where to send the white paper ( #29 ) and which science cases to put in it ( #291 ) - we have some decisions to make as a group! :-)

Please fill out [this doodle poll](http://doodle.com/poll/bxmcx5p92y3ad9fz) by noon PDT tomorrow (Tuesday) and I'll see you on BlueJeans later in the week! 

Cheers

Phil

**Comment by drphilmarshall** [comment]
Thanks for filling out the doodle poll at http://doodle.com/poll/bxmcx5p92y3ad9fz - with 9 out of 20 chapter editors voting, it's a dead heat between Thursday 0800 and 0900 PDT. Vote now! Polls close at noon today :-)

**Comment by AstroVPK** [comment]
I changed my preferences slightly if that helps. I think this favors the
Thursday 0800 timeslot now.

-Vishal

            Vishal Kasliwal
 Dept of Physics & Astronomy,
The University of Pennsylvania
                    and
Dept of Astrophysical Sciences,
        Princeton University

**Comment by drphilmarshall** [comment]
Thanks Vishal - although I hope we are far from a tie break situation! I am
very much hoping that most of the people leading chapters will be able to
make it to the meeting.

**Comment by drphilmarshall** [comment]
Doodle has spoken: we will meet at 0900 PDT on Thursday, April 14th, via BlueJeans. I'll put together an agenda in a Google doc that we can keep notes in, and include connection details. If you have not used BlueJeans before, you can try it out by joining their [demo videocon](http://bluejeans.com/demo).

Before we meet, it would be really helpful if you could prepare to spend two minutes noting in the aforementioned document, and then showing the group, **the sections in your chapter that include a metric analysis** (i.e. ones that at least have metrics defined, and preferably coded and run, too).  I'll share my screen with the current white paper PDF in it, so that we can all see your results. That shortlist of presentable sections will then form the basis for the ensuing discussion of what still needs to be done, what the scope of the paper should be, and where we should be planning on submitting it. I'll send round the notes doc tomorrow. Thanks!

Looking forward to seeing you on Thursday :-)

PS. @rhiannonlynne , I know that you and @davidtrilling cannot make this time: as long as you add your bit to the notes file in advance of Thursday morning, I think @ivezic and I should be able to interpret for you. Thanks! :-)

**Comment by drphilmarshall** [comment]
Oops, sorry - hit send too soon. That should have read:

"prepare, by noting in the aforementioned document, to spend two minutes showing the group, **the sections in your chapter that include a metric analysis**"

The idea is to add pointers to your metrick-ed sections to the notes first, and then show the sections to us all in the meeting. Thanks!

**Comment by drphilmarshall** [comment]
Hi,

I have made us a notes file for our meeting tomorrow. Please find your section and fill in the blanks! :-) I did a very quick audit of the solar system chapter, just to give you an example of what I am after  (great if you could double check this for me, please @rhiannonlynne @davidtrilling )

https://docs.google.com/document/d/1CUTxOnJpiK8eaTC0LriMRlIHD2ig1VObW7lHIzPYwUQ

Note that I am not expecting many sections to be in this state - but once we know which sections *do* actually have quantitative results, we can refine our strategy for finishing v1 of the white paper and getting it submitted. Comments and questions welcome, as always! :-)

Thanks all - looking forward to seeing you tomorrow! :-)

Phil

**Comment by drphilmarshall** [comment]
Hi there - while you figure out the sections you want to point to in our meeting tomorrow, I've added a few notes to provide context for our discussion, and some talking points. Please do take a look at the [notes file](https://docs.google.com/document/d/1CUTxOnJpiK8eaTC0LriMRlIHD2ig1VObW7lHIzPYwUQ). 

@ivezic @egawiser can I also ask you to be prepared to report briefly on the pros and cons of SPIE's Journal of Astronomical Telescopes and Systems (Eric) and PASP (Zeljko)? Thank you!

**Comment by bethwillman** [comment]
Also - If/when there is a crisper version for the scope and depth of the
paper... should consider whether its worth approaching Sarajedini to see
whether AJ Supplements could be a good match.

On Wed, Apr 13, 2016 at 3:32 PM, Phil Marshall <notifications@github.com>
wrote:

> Hi there - while you figure out the sections you want to point to in our
> meeting tomorrow, I've added a few notes to provide context for our
> discussion, and some talking points. Please do take a look at the notes
> file
> <https://docs.google.com/document/d/1CUTxOnJpiK8eaTC0LriMRlIHD2ig1VObW7lHIzPYwUQ>.
>
>
> @ivezic <https://github.com/ivezic> @egawiser
> <https://github.com/egawiser> can I also ask you to be prepared to report
> briefly on the pros and cons of SPIE's Journal of Astronomical Telescopes
> and Systems (Eric) and PASP (Zeljko)? Thank you!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues/293#issuecomment-209654897>
>



-- 
Beth Willman
LSST Deputy Director
Associate Astronomer, Steward Observatory
933 North Cherry Avenue
Tucson, Arizona 85721
520-318-8473

**Comment by AshishMahabal** [comment]
Hi,

I was out of this circuit for the last few days, but hope to attend tomorrows meeting and start syncing.

-ashish

**Comment by drphilmarshall** [comment]
Great, thanks Ashish! Please do take a look at the notes today or early
tomorrow - your typed input is requested! :-)

**Comment by drphilmarshall** [comment]
Morning all! Looking forward to chatting in about an hour. Thanks for all your contributions to the [live notes](https://docs.google.com/document/d/1CUTxOnJpiK8eaTC0LriMRlIHD2ig1VObW7lHIzPYwUQ) so far! I'll take a look at the variables, transients and AGN chapters as well, to make sure we have all bases covered. BlueJeans connection link is [in the notes](https://docs.google.com/document/d/1CUTxOnJpiK8eaTC0LriMRlIHD2ig1VObW7lHIzPYwUQ) - see you there!

**Comment by AshishMahabal** [comment]
Phil, your top-level summary of the Variables section is fine. Will be editing the notes soon. We may not be able to get to individual sections before I need to drop off shortly before 9:30.

-ashish

**Comment by drphilmarshall** [comment]
Dear all,

Thanks very much for your input this morning, some very good ideas came up. I have written up my notes in our [Google document](https://docs.google.com/document/d/1CUTxOnJpiK8eaTC0LriMRlIHD2ig1VObW7lHIzPYwUQ), please do take a few minutes to read through them to make sure I caught everything. Those that could not be there ( including @jasondrhodes @bethwillman @lmwalkowicz @fedhere @willclarkson ), please use the doc's comments to seek clarification, and I will respond. For the crazy busy, here is an executive summary:

* The scale of the current and white paper and the agreed-upon need for peer review push us towards a *series of journal articles* reporting results of metric analyses, starting with one on the simulations and framework itself. 
* The white paper will report to the community via arxiv alone on the Observing Strategy project as a whole, and what we have learned so far. Content from the Observing Strategy repo that it serves to advertize will be processed into journal papers as we go; the living document that is the white paper is simply a compendium of work in progress that seeks to engage and involve the community. This will be explained in the introduction, and to the LSST pub board when we submit it to them.
* We postponed discussion of the paper series to May 27 (when we will submit the WP to the board). The editors are committed to the current white paper timeline, and saw no need to change it.
 
I'll close this issue out, but feel free to carry on posting your reflections! I'll issue our action items next. Great meeting :-)

**Comment by jasondrhodes** [comment]
@drphilmarshall I will discuss the proposed WFIRST SN section with @rubind this week at a WFIRST meeting in Princeton.  I am also still unsure how we want to turn the WFIRST HLS coverage into a FoM.  The real issue is optimal joint timing of the surveys (WFIRST and LSST) as we will eventually get to the full depth in both, by definition.

**Comment by drphilmarshall** [comment]
Hi Chapter Editors,

 Don't forget, we are meeting tomorrow, Thursday April 28, at 0900PDT! I have put a rough agenda into the same [live notes file](https://docs.google.com/document/d/1CUTxOnJpiK8eaTC0LriMRlIHD2ig1VObW7lHIzPYwUQ/edit#heading=h.26drhagt4c0h) that we used last time, please take a look. It has the BlueJeans details in it as well. 

The main thing we will do tomorrow is check our readiness for internal review, so please come prepared with any last minute hold-ups you want to discuss with your fellow editors. I'd like to start issuing reviewing tasks tomorrow afternoon, so that we can make the most of the month of May.

Looking forward to catching up with you - the white paper is looking good, with all your recent work flowing in!

Cheers

Phil

**Comment by jasondrhodes** [comment]
Phil I am out tomorrow through Monday but happy to talk next week.

**Comment by MichelleLochner** [comment]
Apologies, I'm at a workshop! I will look at the minutes and comment after the meeting.

**Comment by AshishMahabal** [comment]
My apologies as well. I am at the "future" surveys meeting in Daejeon welcoming S Korea to LSST and it will be 1 AM here. Beth today and I yesterday told them about the whitepaper and in future versions we will hopefully have contributions from them. 

I will be keeping an eye on the repo. and look forward to the comments. For the variables' chapter we still expect a couple of contributions. Lucianne has said that she plans to be there at the chapter editors' meeting on 28th.

I will be happy to look through the transients chapter later.

**Comment by drphilmarshall** [comment]
Morning all! I just finished most of the version 1 audit ( #304 ) via the PR in #367 . I did a fair bit of reorganisation, so please `git pull` ASAP but also carefully! Hopefully you were not editing your files while I was working on them too (I tried to avoid too many conflicts...) We'll look at my PDF in the meeting in any case. See you soon!

**Comment by drphilmarshall** [comment]
@jasondrhodes @AshishMahabal @bethwillman @MichelleLochner 
Sorry to miss you this morning. [Notes are here](https://docs.google.com/document/d/1CUTxOnJpiK8eaTC0LriMRlIHD2ig1VObW7lHIzPYwUQ/edit#heading=h.26drhagt4c0h) to help you catch up - thanks very much for all the typing, @ebellm ! :-)
